test/nightly/deployWorkspaceProject/testScenarios.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type DeployWorkspaceProjectTestCase } from "./testCases/DeployWorkspaceProjectTestCase";
+import { generateMonoRepoBasicTestCases } from "./testCases/monoRepoBasicTestCases";
+
+interface TestScenario {
```

Comment:
Seems like these test scenarios are not as generic as the name implies. Should it be renamed to `DeployWorkspaceProjectTestScenario`?

I guess what's confusing is that within the generic test scenario, the only property for test cases is specific to deploying a workspace project.

Reply:
Yeah good catch, I should have used that as the name here.